### PR TITLE
Fast review of Specification

### PR DESCRIPTION
**Pull request by frack113**
- Order file in sub-Folder
- Use markdown `\` to add missing line return
- Remove useless attribute schema (no need since correlation move to meta-rules)
- Remove useless file

### CODE REVIEW

File: Sigma_specification.md [hunk 1]
```diff
@@ -627,18 +612,20 @@ Custom placeholders can be defined as required.
 
 # Rule Correlation
 
-// Need to add brief text //
+Correlation allows several events to be linked together. /
+To make it easier to read these corelation rules, they are written in meta-rules.
 
-See [Appendix Mega Rules](appendix_mega_rules.md)
+See [Sigma Mega Rules](Sigma_mega_rules.md)
```
Comment (Res260) [inline Sigma_specification.md:618]:
```suggestion
See [Sigma Meta Rules](Sigma_meta_rules.md)
```

File: Sigma_specification.md [hunk 2]
```diff
@@ -627,18 +612,20 @@ Custom placeholders can be defined as required.
...
 
 # Global filter
 
-// Need to add brief text //
+To adapt the rules to the environment, it is sometimes useful to put the same description in several rules. /
+Their maintaint can become difficult, with a meta-rule it is possible to write it in a single place.
 
-See [Appendix Mega Rules](appendix_mega_rules.md)
+See [Sigma Mega Rules](Sigma_mega_rules.md)
```
Comment (Res260) [inline Sigma_specification.md:625]:
```suggestion
See [Sigma Meta Rules](Sigma_meta_rules.md)
```

File: sigmahq/sigmahq_conventions.md [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
 - [Filenames](#filenames)
+- [Indentation](#indentation)
 - [Titles](#titles)
 - [Status](#status)
+- [Description](#description)
 - [References](#references)
 - [Detection](#detection)
   - [Item Lists](#item-lists)
-- [False Postivess](#false-postivess)
+- [False Postives](#false-postives)
```
Comment (Res260) [inline sigmahq/sigmahq_conventions.md:18]:
```suggestion
- [False Positives](#false-positives)
```

File: Sigma_specification.md [hunk 4]
```diff
@@ -627,18 +612,20 @@ Custom placeholders can be defined as required.
...
+To make it easier to read these corelation rules, they are written in meta-rules.
 
-See [Appendix Mega Rules](appendix_mega_rules.md)
+See [Sigma Mega Rules](Sigma_mega_rules.md)
 
 # Global filter
 
-// Need to add brief text //
+To adapt the rules to the environment, it is sometimes useful to put the same description in several rules. /
+Their maintaint can become difficult, with a meta-rule it is possible to write it in a single place.
```
Comment (Res260) [inline Sigma_specification.md:623]:
```suggestion
Their maintenance can become difficult, with a meta-rule it is possible to write it in a single place.
```

File: Sigma_specification.md [hunk 5]
```diff
@@ -264,34 +244,38 @@ Creator of the rule. (can be a name, nickname, twitter handle...etc)
...
+References to the source that the rule was derived from. \
+These could be blog articles, technical papers, presentations or even tweets.
 
 ## Date (optional)
 
 **Attribute**: date
 
-Creation date of the rule. Use the format YYYY/MM/DD or YYYY-MM-DD
+Creation date of the rule. \
+Use the format YYYY/MM/DD or YYYY-MM-DD
```
Comment (thomaspatzke) [inline Sigma_specification.md:255]: I would prefer to drop the slashed date format completely due to the ambiguity created for people using `YYYY/DD/MM`. `YYYY-MM-DD` is further the ISO standard.

File: Sigma_specification.md [hunk 6]
```diff
@@ -264,34 +244,38 @@ Creator of the rule. (can be a name, nickname, twitter handle...etc)
...
 
 ## Modified (optional)
 
 **Attribute**: modified
 
-*Last* modification date of the rule. Use the format YYYY/MM/DD or YYYY-MM-DD
+*Last* modification date of the rule. \
+Use the format YYYY/MM/DD or YYYY-MM-DD
```
Comment (thomaspatzke) [inline Sigma_specification.md:262]: Same here, discussion above regarding the slashed date format.

**Review by thomaspatzke — CHANGES_REQUESTED** [review]

File: Sigma_specification.md (hunk 5 above)
Comment (frack113) [reply Sigma_specification.md:255]: so "Use the ISO 8601 date with separator format YYYY-MM-DD"
And a small update in the V2_breaking_changes.md

**Review by thomaspatzke — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -15,16 +15,17 @@ The version number is in the form of 3 digits 'A.B.C':
...
+The Sigma Meta-rules format specifications is described in the file [Sigma_meta_rules](Sigma_meta_rules.md)  
 
 There exists other files in the repository to describe the different modifiers, fields and tags to be used in Sigma rules:
 
-- [appendix_modifier](appendix_modifer.md) is a document that defines the different modifiers that can be used in a Sigma rule. 
-- [appendix_tags](appendix_tags.md) is a document that defines the standardized tags that can be used to categorize the different Sigma rules.
-- [appendix_taxonomy](appendix_taxonomy.md) is a document that defines the different field names and log sources that should be used to ensure sharable rules
+- [appendix_modifier](appendix/appendix_modifer.md) is a document that defines the different modifiers that can be used in a Sigma rule. 
+- [appendix_tags](appendix/appendix_tags.md) is a document that defines the standardized tags that can be used to categorize the different Sigma rules.
+- [appendix_taxonomy](appendix/appendix_taxonomy.md) is a document that defines the different field names and log sources that should be used to ensure sharable rules
```
Comment (nasbench) [inline README.md:24]:
```suggestion
- [appendix_taxonomy](appendix/appendix_taxonomy.md) is a document that defines the different field names and log sources that should be used to ensure sharable rules.
```

File: README.md [hunk 8]
```diff
@@ -15,16 +15,17 @@ The version number is in the form of 3 digits 'A.B.C':
...
 
 ## SigmaHQ
 
-The following files are not part of the sigma specification. They are only helpers for the management of the main [rule repository](https://github.com/SigmaHQ/sigma/tree/master/rules)
+The following files are not part of the sigma specification. 
+They are only helpers for the management of the main [rule repository](https://github.com/SigmaHQ/sigma/tree/master/rules)
```
Comment (nasbench) [inline README.md:29]:
```suggestion
They are only helpers for the management of the main [rule repository](https://github.com/SigmaHQ/sigma/tree/master/rules) under SigmaHQ
```

File: Sigma_specification.md [hunk 9]
```diff
@@ -461,14 +445,14 @@ Examples ii:
 * `<Data Name="ServiceName">MpKsl4eaa0a76</Data>` will be `ServiceName`
 
 ### Special Field Values
-
+/// NEED REVIEW
+modifier `exists` ?
+///
```
Comment (nasbench) [inline Sigma_specification.md:450]: What is this section referring to?

File: Sigma_specification.md [hunk 10]
```diff
@@ -592,10 +576,11 @@ A Sigma rule can be categorised with tags. Tags should generally follow this syn
 * Keep tags short, e.g. numeric identifiers instead of long sentences
 * Feel free to send pull request or issues with proposals for new tags
 
-[More information about tags](appendix_tags.md)
+[More information about tags](appendix/appendix_tags.md)
 
 ## Placeholders
-
+/// NEED REVIEW
+///
```
Comment (nasbench) [inline Sigma_specification.md:583]: to be removed

**Review by nasbench — APPROVED** [review]

File: Sigma_specification.md (hunk 9 above)
Comment (frack113) [reply Sigma_specification.md:450]: normally `not null` should be written as `|exists: true` now , and a null value `|exists: false` (I think).
But empty on the other hand is always `: ''`.

File: Sigma_specification.md (hunk 10 above)
Comment (frack113) [reply Sigma_specification.md:583]: Even if Placeholders section look goot to me , is it good ?

**Pull request merged**